platformio.ini

```diff
@@ -21,5 +21,3 @@ build_unflags = -std=gnu++11
 build_flags =
     -std=c++14
     -fmax-errors=5
-
-targets = upload
```

Comment:
Is this change OK @Tasssadar? I need to delete this line due to [Travis build](https://travis-ci.org/RoboticsBrno/RB3201-RBControl-library/jobs/475515866#L662).

When I made a quick look at the [PIO documentation](https://travis-ci.org/RoboticsBrno/RB3201-RBControl-library/jobs/475515866#L662) I considered that we don't need it. 

Am I correct?